### PR TITLE
updateQuery of subscribeToMore should be optional

### PR DESCRIPTION
**Pull request by Torsten85**
As mentioned in #871, the updateQuery function can no be covered with the new graphql options.reducer. So it should be optional.

### CODE REVIEW

File: src/core/ObservableQuery.ts [hunk 1]
```diff
@@ -228,7 +228,7 @@ export class ObservableQuery extends Observable<ApolloQueryResult> {
       variables: options.variables,
     });
 
-    const reducer = options.updateQuery;
+    const reducer = options.updateQuery || ((previousResult) => previousResult);
```
Comment (Torsten85) [inline src/core/ObservableQuery.ts]: Wouldn't the complete `next` function be obsolete if `updateQuery` is not passed? Seems like `next` is optional.

https://github.com/apollostack/apollo-client/blob/master/src/util/Observable.ts#L38

**Comment by Torsten85** [comment]
@stubailo could you have another look.

**Comment by helfer** [comment]
@Torsten85 I can review it, but can you merge in the changes from master first? We added a couple of lines of code there.

**Comment by Torsten85** [comment]
This pull request should now be up to date with the current master branch.

**Comment by helfer** [comment]
@Torsten85 I think it's missing a test that shows that `updateQueries` really is optional, and that everything works as intended when `updateQueries` isn't provided. Other than that, it looks fine 🙂

**Comment by helfer** [comment]
@Torsten85 are you still interested in this PR? 🙂

**Comment by helfer** [comment]
I'm trying to put some order into issues and PRs, so I'm going to close this one for lack of activity, but that doesn't mean that I'm not interested in seeing the feature implemented eventually.

**Pull request closed without merging**